### ISSUE TITLE
Ability to push the custom built instaslice-operator images to OCP cluster

### ISSUE THREAD

**Issue by sairameshv** [openshift/instaslice-operator#45]
As of today, we have an ability to build the instaslice operator images( i.e. controller and the daemon set images) from the custom code.

We should now have an ability to update the existing openshift cluster with these custom images mainly for the purpose of the development.

**Comment by asm582** [comment]
@sairameshv are we still working on this design to push images?

**Comment by sairameshv** [comment]
Yes @asm582 , I have learnt a way to achieve this, but I'm exploring on a different way from CI side to build and push the images to the CI Image registry to leverage those in our e2e tests and CI jobs

**Comment by asm582** [comment]
@sairameshv are we still working on this issue?

**Comment by sairameshv** [comment]
We can close this issue as we have a way to pull/push the images from/to quay.io
